samples/NHSD.BuyingCatalogue.Identity.Api.SampleMvcClient/Views/Home/Privacy.cshtml

```diff
@@ -5,3 +5,18 @@
 
 <p>You are successfully logged in</p>
 <p id="sampleResourceResult">@ViewBag.Response</p>
+
```

Comment:
If we have a separate page for this, is there a reason to add it here too?

Reply:
There are two sets of claims. ID claims & user claims. Hence the introduction of a new page as need to actually go to the Identity server to retrieve the user claims .